src/product.jl

```diff
@@ -203,6 +209,9 @@ end
 function FusionStyle(::Type{<:ProductSector{T}}) where {T <: SectorTuple}
     return Base.:&(map(FusionStyle, _sectors(T))...)
 end
+function UnitStyle(::Type{<:ProductSector{T}}) where {T <: SectorTuple}
+    return Base.:&(map(UnitStyle, _sectors(T))...)
```

Comment:
```suggestion
    return mapreduce(UnitStyle, &, _sectors(T))
```
I'm not sure why the other functions aren't defined in the same way, @Jutho do you know if this is because that doesn't work (or used to?) with tuples and type stability?

Reply:
That could be yes, there was a time where `map` over heterogenous tuples was not type stable.